### PR TITLE
Grace/nav

### PR DESCRIPTION
**Pull request by gracesnoh**
image

image

### CODE REVIEW

**Comment by caseytakahashi** [comment]
Some new requests from Kris & Leah:

1. On the landing page, can we make the nav bar stay blue over the map instead of translucent? 
screen shot 2017-04-05 at 2 19 45 pm

2. Also, they want to increase the amount that the 2nd section below the map peeks out from beneath. I think on other computer screen sizes, it doesn't show up (they can't see that it peeks out) so let's increase the size of the white beneath the map!

File: app/views/partials/_nav.html.erb [hunk 1]
```diff
@@ -59,37 +59,26 @@
   <div class="nav">
     <div class="container-lg">
       <a href="/" class="logo"></a>
-
-      <form class="discussion-search">
```
Comment (caseytakahashi) [inline app/views/partials/_nav.html.erb]: They still want to include search in the discussions page! just not have it in the nav bar - so I think the best spot is in the light blue section (make the light blue section a bit thicker and add in the search bar above where the tags are)

screen shot 2017-04-05 at 2 24 25 pm

**Review by caseytakahashi — CHANGES_REQUESTED** [review]

**Comment by gracesnoh** [comment]
## Blue Nav + More Second Section
image

## Responsive Nav
image
image
image

**Comment by caseytakahashi** [comment]
OOOH this looks so good and love the way you styled the nav bar when it's on mobile :) LGTM!

**Review by caseytakahashi — APPROVED** [review]

**Pull request merged**